src/Okta.Sdk.IntegrationTests/GroupScenarios.cs

```diff
@@ -119,10 +119,224 @@ public class GroupScenarios : ScenarioGroup
                 () => client.Groups.GetGroupAsync(createdGroup.Id));
         }
 
-        [Fact(Skip = "TODO")]
-        public async Task GroupUserOperations()
+        [Fact]
+        public async Task AddUserToGroup()
         {
-            throw new NotImplementedException();
+            var client = GetClient();
+
+            // Create group
+            var createdGroup = await client.Groups.CreateGroupAsync(new CreateGroupOptions
+            {
+                Name = "New Users Test Group",
+            });
+
+            // Create a user
+            var createdUser = await client.Users.CreateUserAsync(new CreateUserWithPasswordOptions
+            {
+                Profile = new UserProfile
+                {
+                    FirstName = "John",
+                    LastName = "Get-User",
+                    Email = "john-get-user@example.com",
+                    Login = "john-get-user@example.com",
+                },
+                Password = "Abcd1234",
+                Activate = true,
+            });
+
+            await client.Groups.AddUserToGroupAsync(createdGroup.Id, createdUser.Id);
```

Comment:
I think these lines should also be within the `try`. That way if it fails at the beginning, things are still cleaned up. What do you think? (applies to all these new tests)

Reply:
if I want `createdUser` to be available in the `finally` scope, it should be declared before `try` block
@nbarbettini

Reply:
I should have been more specific - I meant just the `AddUserToGroup` line. You understood what I meant 🙂  👍